graph/util/Encoding.java

```diff
@@ -285,49 +217,46 @@ public byte key() {
      *
      * The size of a prefix is 1 byte; i.e. min-value = 0 and max-value = 255.
      */
-    // TODO I think we should compress these more
+    // TODO we should compress these more
     // TODO for example, group properties 1-50, type edges 40-60, instance edges 60-80, reserved for future use 80-127
```

Comment:
Do you think we should act on this now? while we can still re-release the first alpha?

Reply:
fair point, will do